sdk/messaging/azservicebus/stress/stress.go

```diff
@@ -85,37 +85,42 @@ func runBasicSendAndReceiveTest() {
 	cleanupQueue := createQueue(telemetryClient, cs, queueName)
 	defer cleanupQueue()
 
+	ctx, cancel := context.WithTimeout(context.Background(), time.Hour*24*5)
```

Comment:
Time intervals are usually written as "value * unit" so that it reads naturally, i.e. `5*24*time.Hour`

Reply:
Done